datahub-web-react/src/app/shared/tags/CreateTagModal.tsx

```diff
@@ -48,13 +40,11 @@ export default function CreateTagModal({
         })
             .then(() => {
                 // then apply the tag to the dataset
-                addTagMutation({
```

Comment:
what happens if a user selects 3 existing tags to add and then types in a fourth tag name, finds it doesnt exist, and then creates it? Will those other 3 tags just be dropped? wondering if we should handle this case either by preventing it or making sure those other 3 tags are added as well.

Reply:
We only allow creation on the first tag :)